android/app/src/main/res/values/ic_launcher_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#141414</color>
+</resources>
```

Comment:
There are some files missing empty line at the end of the file - should it be left like that or fixed?

Reply:
Most files inside `android` folder are autogenerated, I don't think it adds much value to manually add empty lines to them every time

Reply:
Sure, if they are generated automatically there's no point in changing them